core/templates/components/forms/custom-forms-directives/image-uploader.directive.ts

```diff
@@ -48,7 +48,7 @@ angular.module('oppia').directive('imageUploader', [
             'image-uploader-is-active');
         };
 
-        var validateUploadedFile = function(file, filename) {
+        var validateUploadedFile = function(file, _filename) {
```

Comment:
Ditto here and elsewhere.

Reply:
Used underscore here to allow unused argument.